public/common/stylesheets/color-palette.styl

```diff
@@ -17,3 +15,18 @@ dark-gold = #bf9b30
 
 fog-60 = rgba(255, 255, 255, 0.6)
 fog-80 = rgba(255, 255, 255, 0.8)
+
+
+holy-white = #fff
+
+background-green-0 = #183d3d
+background-green-1 = #224b4b
+background-green-2 = #2d5d59
+background-green-3 = #43827f
+game-background = linear-gradient(to top, background-green-0 0%, background-green-1 35%, background-green-2 70%, background-green-3 100%)
```

Comment:
`game-background` could be moved to `public/common/stylesheets/common.styl`, OK?